vivekrnv/sonic-linux-kernel
# [202311] [Mellanox] Integrate HW-MGMT 7.0030.4001

#### What I did
- Integrate new hw-mgmt changes
- Add support for compiling linux kernel with custom platform patches and kconfig

#### How to Verify
- Compile linux kernel for mellanox and vs platforms. Make sure compilation is succeeded
- Run regression tests on mellanox platform
